test/support/connection_test_helper.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Janus.ConnectionTest.Helper do
+  alias Janus.Connection
+  alias FakeHandler.Payloads
+
+  def handlers_last_callback_call({:state, _, _, _, %{callback: callback}, _}), do: callback
```

Comment:
I would just rework this as "assert_last_callback_called" and inject assert with pattern matching.